apps/web/middleware.ts

```diff
@@ -166,36 +166,28 @@ export const config = {
     "/api/auth/signup",
     "/api/trpc/:path*",
     "/login",
-    "/auth/login",
-    "/future/auth/login",
-    /**
-     * Paths required by routingForms.handle
-     */
-    "/apps/routing_forms/:path*",
-
-    "/event-types",
-    "/future/event-types/",
-    "/apps/installed/:category/",
-    "/future/apps/installed/:category/",
-    "/apps/:slug/",
-    "/future/apps/:slug/",
-    "/apps/:slug/setup/",
-    "/future/apps/:slug/setup/",
-    "/apps/categories/",
-    "/future/apps/categories/",
-    "/apps/categories/:category/",
-    "/future/apps/categories/:category/",
+    "/auth/:path*",
+    "/apps/:path*",
+    "/event-types/:path*",
     "/workflows/:path*",
     "/getting-started/:path*",
     "/apps",
     "/bookings/:status/",
     "/video/:path*",
     "/teams",
-    "/future/teams/",
+    "/signup",
     "/settings/:path*",
     "/reschedule/:path*",
     "/availability/:path*",
     "/booking/:path*",
+    "/org/:path*",
+    "/team/:path*",
+    "/booking/:path*",
+    "/payment/:path*",
+    "/router/:path*",
+    "/routing-forms/:path*",
+    "/:user/:type/",
+    "/:user/",
```

Comment:
Why the change here?

- Our new 404 page in App Router needs to use custom headers set in middleware (e.g., `x-pathname`, `x-forwarded-host`, `x-locale`).
- Hence, we need to ensure that every route goes through middleware so that the 404 page works as expected.